Conditions de explorization

h-kataria opened the pull request:
Dependent core [PR](https://github.com/ManageIQ/manageiq/pull/20948)
Data migration [PR](https://github.com/ManageIQ/manageiq-schema/pull/551)

skateman commented on app/helpers/application_helper/toolbar_chooser.rb:
```diff
@@ -35,7 +35,7 @@ def view_toolbar_filename
     elsif %w[container_project].include?(@layout)
       'container_project_view_tb'
-    elsif !%w[all_tasks timeline diagnostics my_tasks miq_server usage].include?(@layout) &&
+    elsif !%w[all_tasks condition timeline diagnostics my_tasks miq_server usage].include?(@layout) &&
```
Can't we use the declarative way with the `toolbar` method in the controller?

skateman commented on app/views/condition/show.html.haml:
```diff
@@ -0,0 +1 @@
+= render :partial => 'condition_details'
```
Please move the view's content just here...

skateman replied:
^^ Having a single file referencing to a partial

skateman reviewed (changes requested):
+rebase

h-kataria commented:
> @h-kataria I ran into these issues with my initial testing of these changes
> 
>     * [ ]  Edit is available for a read-only condition when selected from the show list screen
> 
>     * [ ]  No option to copy a condition from the show list screen. Also, it's impossible to copy a read-only condition because the `Configuration` menu is disabled when in the detail screen if the condition is read-only
> 
>     * [ ]  Getting a 404 when selecting any option on the `Download` menu

I am working on Edit button from list view, in previous version there were no toolbars on list view.

As for Condition copy button on summary screen, that is the same behavior as in previous version, Copy for read only conditions is not allowed.

Fixed Download button issue, Download button was not available on Explorer version, so i removed that from non-explorer version.

h-kataria commented:
> @h-kataria I ran into these issues with my initial testing of these changes
> 
>     * [ ]  Edit is available for a read-only condition when selected from the show list screen - Fixed

![Screenshot from 2021-02-03 13-29-47](https://user-images.githubusercontent.com/3450808/106792307-f7ab0180-6623-11eb-8ee2-884debb0ef8f.png)

> 
>     * [ ]  No option to copy a condition from the show list screen. Also, it's impossible to copy a read-only condition because the `Configuration` menu is disabled when in the detail screen if the condition is read-only
> 
>     * [ ]  Getting a 404 when selecting any option on the `Download` menu

h-kataria commented:
@gtanzillo please re-test, addressed your comments above

gtanzillo reviewed (approved):
This looks good., tested locally 👍

skateman reviewed (approved):
![The Seal of Approval](http://www.skateman.eu/seal.jpg)
